src/App.tsx

```diff
@@ -1,14 +1,53 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [userId, setUserId] = useState<number>(0);
+  const [query, setQuery] = useState('');
```

Comment:
If you give type to number - add to the string
But it's unnecessary, bc typescript already know about type if you put initial value
```suggestion
  const [userId, setUserId] = useState(0);
  const [query, setQuery] = useState('');
```

```suggestion
  const [userId, setUserId] = useState<number>(0);
  const [query, setQuery] = useState<string>('');
```